Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/refreshable-submission-list/RefreshableSubmissionList.tsx

```diff
@@ -0,0 +1,117 @@
+    const { state: { user: { permissions: { canAccessAdministration } } } } = useAuth();
+    const dispatch = useDispatch();
+    const { currentSubmission, currentSubmissionResults } =
+    useSelector((state: any) => state.submissionDetails);
+    const handlePageChange = useCallback(
+        (page: number) => {
+            dispatch(setCurrentPage(page));
+        },
+        [ dispatch ],
+    );
+    const handleReloadClick = useCallback(
+        async () => {
+            reload();
+        },
+        [ reload ],
+    );
+    const renderButtonsSection = useCallback(() => (
+        <div className={styles.buttonsSection}>
+            <Button
+              onClick={handleReloadClick}
+              text="Reload"
+              type={ButtonType.secondary}
+              className={styles.submissionReloadBtn}
+            />
+            {renderRetestButton()}
+        </div>
+    ), [ handleReloadClick, renderRetestButton ]);
+
+    const renderSubmissionInfo = useCallback(
+        () => {
+            if (!canAccessAdministration || isNil(currentSubmission)) {
+                return null;
+            }
+
+            const { createdOn, modifiedOn, startedExecutionOn, completedExecutionOn, user: { userName } } = currentSubmission;
+
+            return (
+                <div className={styles.submissionInfo}>
+                    <p className={styles.submissionInfoParagraph}>
+                        Created on:
+                        {' '}
+                        {preciseFormatDate(createdOn)}
+                    </p>
+                    <p className={styles.submissionInfoParagraph}>
+                        Modified on:
+                        {' '}
+                        {isNil(modifiedOn)
+                            ? 'never'
+                            : preciseFormatDate(modifiedOn)}
+                    </p>
+                    <p className={styles.submissionInfoParagraph}>
+                        Started execution on:
+                        {' '}
+                        {isNil(startedExecutionOn)
+                            ? 'never'
+                            : preciseFormatDate(startedExecutionOn)}
+                    </p>
+                    <p className={styles.submissionInfoParagraph}>
+                        Completed execution on:
+                        {' '}
+                        {isNil(completedExecutionOn)
+                            ? 'never'
+                            : preciseFormatDate(completedExecutionOn)}
+                    </p>
+                    <p className={styles.submissionInfoParagraph}>
+                        Username:
+                        {' '}
+                        {userName}
+                    </p>
+                </div>
+            );
+        },
+        [ currentSubmission, canAccessAdministration ],
+    );
+
+    return (
+        <div className={styles.navigation}>
+            <div style={{ marginBottom: '24px' }}>
```

Comment:
Can this inline styling be moved in the css files?

Reply:
With the possibility of scale, we could finish with hundreds of classes which contains just one property. Should i move it to css file ?

Reply:
Inline styles should be generally revised.